Shopist/ShopistUITests/ShopistUITests.swift

```diff
@@ -1,18 +1,57 @@
 /*
-* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
-* This product includes software developed at Datadog (https://www.datadoghq.com/).
-* Copyright 2019-2020 Datadog, Inc.
-*/
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
 
 import XCTest
 
 class ShopistUITests: XCTestCase {
-    override func setUp() {
+    override func setUp() { // swiftlint:disable:this empty_xctest_method
         continueAfterFailure = false
     }
 
-    func testAppCanBeLaunched() {
+    func testRunFullCheckoutFlow() {
+        let sessionCount = 3
         let app = XCUIApplication()
-        app.launch()
+        for sessionIndex in 0...sessionCount {
```

Comment:
Why running it 4 times?

Reply:
4 is only for debugging, it will be 20 something when we run them regularly.
android runs the same scenarios many times to generate multiple sessions, i don't know if they do it in source code or Gradle config but this is what @xgouchet told me.

Reply:
One Bitrise job runs around 100 scenarios (each one takes around 20 secs to run on average). This allows us to "fill" 30 minutes of data for each run.

Reply:
I see 👌. Then we must check how does it behave in case of failure. Currently we set:
```swift
continueAfterFailure = false
```
which means "stop UITests on the first failure". So if the 1st iteration fails, no more data from given CI run will be send to demo org (leaving the gap in demo fixture).

Reply:
as the demo data must be consistent all the time, if anything goes wrong shouldn't we discard the run? a previous batch will be used at worst case. if we continue after failure, inconsistent data will be sent to demo org